containers/Dockerfile.buildenv

```diff
@@ -38,8 +38,29 @@
 # If you do not have permission to push to DockerHub please coordinate with
 # someone who does. Since changes to this image effect our testing
 # infrastructure it is important all changes be carefully reviewed.
+#
+# NOTE: If building for Apple Silicon, then add the following arguments to
+# the docker build command in step 1 and in step 3 (building the ci target)
+# is not currently supported, since GitHub actions do not yet support
+# Apple Silicon:
```

Comment:
Even though CI can't support Apply Silicon, should we still modify the CI image to work with arm so it's still usable? Some devs may want to run with different compilers on arm and this would be how to do it.

Reply:
Would be nice but don't need to do it in this PR I think. I don't see a use case right now.

Reply:
Ok sounds good.